### DISCUSSION TITLE
Add Ultra low latency option

### DISCUSSION THREAD

**Discussion by developomp** [NVIDIA/open-gpu-kernel-modules#620] [Ideas]
On Windows, we can set low latency mode to "Ultra" to get the lowest amount of render queue latency at a driver level.

> ... we’re introducing a new Ultra-Low Latency Mode that enables ‘just in time’ frame scheduling, submitting frames to be rendered just before the GPU needs them ...
>
> from https://www.nvidia.com/en-us/geforce/news/gfecnt/gamescom-2019-game-ready-driver/

On Linux, however, the best thing we can do is to set `__GL_MaxFramesAllowed=1` which is only equivalent to "Low Latency: on" on Windows.

Bringing this feature to Linux would greatly improve its viability as a competitive gaming platform.

**Comment by somesillydude69** [comment]
where do i even put __GL_MaxFramesAllowed=1?

**Reply by edisionnano** [reply]
/etc/environment

**Comment by Freddycat** [comment]
I really hope this gains more traction, this should really be a thing on linux!
